Repair laser with correct damage

RuneThe opened the pull request.

RuneThe committed:
repairing Lazer.class

RuneThe committed:
Fixed laser, added two tests, all working

petterdaae reviewed (commented):
There is one test failing in RoboTests now. Can you fix it before merging to master?

RuneThe commented:
Checking robot tests now

RuneThe committed:
Fixed Robot.class so no robotTests feil now

RuneThe commented:
Fixed robot class, as takeAnergy was incorrect in efforts to fix laser.class. Now corrected. All tests run well.
Planning more test of laser-robot-board-laserEvent
